homeassistant/components/kostal_plenticore/config_flow.py

```diff
@@ -21,6 +21,7 @@
     {
         vol.Required(CONF_HOST): str,
         vol.Required(CONF_PASSWORD): str,
+        vol.Optional(SERVICE_CODE): str,
```

Comment:
Let's rename it to CONF_SERVICE_CODE and make it a snake_case string and provide the name via translations